VAMobile/e2e/tests/AvailabilityFramework.e2e.ts

```diff
@@ -11,26 +13,183 @@ const AFNavigation = [
     'How to update or fix an error in your legal name',
   ],
   ['FullScreenSubtask', 'WG_PreferredName', 'Profile', 'Personal information', 'Preferred name'],
-  //['CategoryLanding', 'WG_Health', 'Health'],
 ]
 
-describe(':android: Availability Framework', () => {
-  for (let x = 0; x < AFNavigation.length; x++) {
-    it('should verify AF use case 1 for: ' + AFNavigation[x][0], async () => {
-      if (AFNavigation[x][1] !== 'WG_Health') {
-        await enableAF(AFNavigation[x][1], 'DenyAccess')
-        await verifyAF(AFNavigation[x], 'DenyAccess')
-      }
-    })
+const AFNavigationForIndividual = [
+  ['PersonalInformationScreen.e2e', 'WG_PersonalInformation', 'Profile', 'Personal information'],
+  [
```

Comment:
It is organized by the waygate order in the dev settings (minus one or two that may be out of order because I re-added them in last minute). I had it that way originally so I could easily tell if I was missing a waygate and kept it that way when I added the script names.  I can see arguments for having it either way though.